### ISSUE TITLE
emoji size changes in adapter

### ISSUE THREAD

**Issue by martipello** [vanniktech/Emoji#284]
Thank you for filing an issue. If this is a bug that you want to report, please take the time to provide some information:

- Version of the library:  implementation 'com.vanniktech:emoji-google:0.5.1'
- Affected devices:  all android devices ive tested from 4.4.2 to 8.0

Screenshot(s) showing the issue and reprodution steps are appreciated.

I'm using the com.vanniktech.emoji.EmojiTextView in an adapter in the same way as your sample I've also added a method that allows me to select views but when i select and deselect specific views within the adapter the emojis change size irregularly, my select and deselect only make a view (outside emoji texts heirarchy) visible and invisible, here is the pertinant code i have in my onBindViewHolder

if (userMessage.getMessage().length() < 1){
            holder.message_text.setVisibility(View.GONE);
            System.out.println("message text is empty : " + userMessage.getMessage());
        }
        else if (userMessage.getMessage().length() < 10){
            holder.message_text.setTextSize(22f);
            System.out.println("less than ten characters : " + userMessage.getMessage());
        }
        else{
            holder.message_text.setTextSize(14f);
            System.out.println("more than ten characters : " + userMessage.getMessage());
        }
        switch (userMessage.getSentReceived()){
            case 0 :
                holder.sentReceived.setBackground(context.getResources().getDrawable(R.drawable.circle_white));
                break;
            case 1 :
                holder.sentReceived.setBackground(context.getResources().getDrawable(R.drawable.circle_grey));
                break;
            case 2 :
                holder.sentReceived.setBackground(context.getResources().getDrawable(R.drawable.circle_blue));
                break;

        }
        final EmojiInformation emojiInformation = EmojiUtils.emojiInformation(userMessage.getMessage());
        final int res;

        if (emojiInformation.isOnlyEmojis && emojiInformation.emojis.size() == 1) {
            res = R.dimen.emoji_size_single_emoji;
            System.out.println("one lonely emoji " + userMessage.getMessage());

        } else if (emojiInformation.isOnlyEmojis && emojiInformation.emojis.size() > 1) {
            res = R.dimen.emoji_size_only_emojis;
            System.out.println("one lonely emoji " + userMessage.getMessage());
        } else {
            res = R.dimen.emoji_size_default;
            System.out.println("one lonely emoji " + userMessage.getMessage());
        }
        holder.message_text.setEmojiSizeRes(res, false);

        if (selectedItems.get(position, false)) {
            holder.fade.setVisibility(View.VISIBLE);
        } else {
            holder.fade.setVisibility(View.INVISIBLE);
        }

here is what this prints on first load ([image 1])(https://www.dropbox.com/s/6rp3sxgxs84sp81/Screenshot_2018-06-08-19-43-00.png?dl=0)

 I/System.out: less than ten characters : Hi
 I/System.out: one lonely emoji Hi
 I/System.out: less than ten characters : What's up
 I/System.out: one lonely emoji What's up
 I/System.out: less than ten characters : ��
 I/System.out: one lonely emoji ��
 I/System.out: more than ten characters : Lets send a message longer than ten characters
 I/System.out: one lonely emoji Lets send a message longer than ten characters
 I/System.out: less than ten characters : Hey
 I/System.out: one lonely emoji Hey
 D/MA-sUS: State Set: {"status":"Available"}
 I/System.out: more than ten characters : Why does this have issues
 I/System.out: one lonely emoji Why does this have issues
 I/System.out: more than ten characters : Why does this have issues
 I/System.out: one lonely emoji Why does this have issues
 I/System.out: more than ten characters : Lets send a message longer than ten ��characters 
 I/System.out: one lonely emoji Lets send a message longer than ten ��characters 
 I/System.out: less than ten characters : Hi ��
 I/System.out: one lonely emoji Hi ��
 V/FA: Inactivity, disconnecting from the service

and here is what it prints if i select one view with an emoji and then another ([image 2])(https://www.dropbox.com/s/6rp3sxgxs84sp81/Screenshot_2018-06-08-19-43-00.png?dl=0)

 
 I/System.out: less than ten characters : ��
 I/System.out: one lonely emoji ��
 I/System.out: more than ten characters : Lets send a message longer than ten ��characters 

so to summarise the print outs and the code seems to be all fine but you can see in the images that for some reason the emoji size changes when (according to the logs) it shouldn't be any ideas?

----------------------------------------------------------------------

**Comment by vanniktech** [comment]
Sorry but this report is not actionable. Formatting is super off and I don't understand exactly what's going on. I'll refer to the sample app as I believe it has what you want.

**Comment by martipello** [comment]
Fixed formatting please have a look, as stated I am using the sample app
